src/Mono.Android/Properties/AssemblyInfo.cs.in

```diff
@@ -27,6 +27,7 @@ using System.Runtime.CompilerServices;
 [assembly: System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(System.Drawing.Size))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(System.Drawing.SizeF))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(System.Drawing.SystemColors))]
+[assembly: InternalsVisibleTo("Mono.Android.Export, PublicKey=0024000004800000940000000602000000240000525341310004000011000000438ac2a5acfbf16cbd2b2b47a62762f273df9cb2795ceccdf77d10bf508e69e7a362ea7a45455bbf3ac955e1f2e2814f144e5d817efc4c6502cc012df310783348304e3ae38573c6d658c234025821fda87a0be8a0d504df564e2c93b2b878925f42503e9d54dfef9f9586d9e6f38a305769587b1de01f6c0410328b2c9733db")]
```

Comment:
Why is this change needed? Does the C# compiler "know about" `[DynamicDependency]` and require visibility to the specified member?

Reply:
Because `DynamicDependencyAttribute` is internal attribute in `Mono.Android` for monodroid target framework - that's our "placeholder" to avoid `#if`'s .